### DISCUSSION TITLE
Sugar Valley NeoPool Controller

### DISCUSSION THREAD

**Discussion by denzel1994** [arendst/Tasmota#12163] [Q&A] [answered]
Hey guys,

does anyone have this build complete  [https://tasmota.github.io/docs/NeoPool/](url) and has a picture of the correct wiring including the TTL UART to RS485 converter?

Thank you guys for the awesome support.

Lars

**Comment by sfromis** [comment, marked as answer]
Not me, but #10795 also has discussion around RS485 wiring.
